vcam-util.c

```diff
@@ -126,10 +134,13 @@ int modify_device(struct vcam_device_spec *dev)
         dev->pix_fmt = orig_dev.pix_fmt;
 
     int res = ioctl(fd, VCAM_IOCTL_MODIFY_SETTING, dev);
-    printf("Setting modified\n");
+    if (res) {
+        fprintf(stderr, "Failed to modify the device.\n");
+    } else {
+        fprintf(stdout, "Modified the device successfully.\n");
```

Comment:
For error messages, we would use stderr. Otherwise, take stdout. By the way, the message ""Modified the device successfully" seems to be annoying. Let's get it out.